DigitalPassaport/DigitalPassaport/Epics/CreateAccount/Repository/CreateAccountRepository.swift

```diff
@@ -15,11 +15,14 @@ protocol CreateAccountRepository {
 class CreateAccountRepositoryImpl: CreateAccountRepository {
     
     private let networkingService: NetworkingService
+    private let coreDataManager: CoreDataManager
     
-    init(networkingService: NetworkingService) {
+    init(networkingService: NetworkingService, coreDataManager: CoreDataManager) {
```

Comment:
If already use singleton you can set default value here = CoreDataManager.shared